.strict-typing

```diff
@@ -268,6 +268,7 @@ homeassistant.components.tile.*
 homeassistant.components.tilt_ble.*
 homeassistant.components.tolo.*
 homeassistant.components.tplink.*
+homeassistant.components.tplink_omada.*
```

Comment:
Can you add a `brands` file for TP-Link to group the TP-Link integrations

Reply:
Done